sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/PartitionKeyHelper.java

```diff
@@ -47,7 +47,7 @@ public static PartitionKey extractPartitionKeyFromDocument(
                             return ImplementationBridgeHelpers
```

Comment:
We can remove `getPartitionKeyObjectKey` - since it anyways has the unsupported cast logic.

Reply:
removed